### ISSUE TITLE
[Bug] On first join (Folia 1.21.1)

### ISSUE THREAD

**Issue by mani1232** [Syrent/SayanVanish#332]
```
[13:43:30 INFO]: UUID of player mani1232 is b8a29be6-861c-453a-93e5-eaa0ddc9f962
[13:43:32 ERROR]: Could not pass event BukkitUserVanishEvent to SayanVanish v1.4.1-SNAPSHOT-build.175-e080e05
java.lang.NullPointerException: null
        at SayanVanish-1.4.1-SNAPSHOT-build.175-e080e05-bukkit.jar/org.sayandev.sayanvanish.lib.stickynote.bukkit.nms.PacketUtils.getMobEffectByEffectType(PacketUtils.kt:154) ~[SayanVanish-1.4.1-SNAPSHOT-build.175-e080e05-bukkit.jar:?]
        at SayanVanish-1.4.1-SNAPSHOT-build.175-e080e05-bukkit.jar/org.sayandev.sayanvanish.lib.stickynote.bukkit.nms.PacketUtils.getMobEffectPacket(PacketUtils.kt:129) ~[SayanVanish-1.4.1-SNAPSHOT-build.175-e080e05-bukkit.jar:?]
        at SayanVanish-1.4.1-SNAPSHOT-build.175-e080e05-bukkit.jar/org.sayandev.sayanvanish.lib.stickynote.bukkit.nms.PacketUtils.getUpdateMobEffectPacket(PacketUtils.kt:164) ~[SayanVanish-1.4.1-SNAPSHOT-build.175-e080e05-bukkit.jar:?]
        at SayanVanish-1.4.1-SNAPSHOT-build.175-e080e05-bukkit.jar/org.sayandev.sayanvanish.bukkit.feature.features.FeatureEffect.onVanish(FeatureEffect.kt:60) ~[SayanVanish-1.4.1-SNAPSHOT-build.175-e080e05-bukkit.jar:?]
        at com.destroystokyo.paper.event.executor.MethodHandleEventExecutor.execute(MethodHandleEventExecutor.java:40) ~[paper-mojangapi-1.21.1-R0.1-SNAPSHOT.jar:?]
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[paper-mojangapi-1.21.1-R0.1-SNAPSHOT.jar:1.21.1-DEV-cc907e8]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[paper-mojangapi-1.21.1-R0.1-SNAPSHOT.jar:?]
        at io.papermc.paper.plugin.manager.PaperEventManager.callEvent(PaperEventManager.java:56) ~[luminol-1.21.1.jar:1.21.1-DEV-cc907e8]
        at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.callEvent(PaperPluginManagerImpl.java:131) ~[luminol-1.21.1.jar:1.21.1-DEV-cc907e8]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:628) ~[paper-mojangapi-1.21.1-R0.1-SNAPSHOT.jar:?]
        at SayanVanish-1.4.1-SNAPSHOT-build.175-e080e05-bukkit.jar/org.sayandev.sayanvanish.bukkit.api.BukkitUser.vanish(BukkitUser.kt:57) ~[SayanVanish-1.4.1-SNAPSHOT-build.175-e080e05-bukkit.jar:?]
        at SayanVanish-1.4.1-SNAPSHOT-build.175-e080e05-bukkit.jar/org.sayandev.sayanvanish.bukkit.feature.features.FeatureState.onJoin(FeatureState.kt:64) ~[SayanVanish-1.4.1-SNAPSHOT-build.175-e080e05-bukkit.jar:?]
        at com.destroystokyo.paper.event.executor.MethodHandleEventExecutor.execute(MethodHandleEventExecutor.java:40) ~[paper-mojangapi-1.21.1-R0.1-SNAPSHOT.jar:?]
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[paper-mojangapi-1.21.1-R0.1-SNAPSHOT.jar:1.21.1-DEV-cc907e8]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[paper-mojangapi-1.21.1-R0.1-SNAPSHOT.jar:?]
        at io.papermc.paper.plugin.manager.PaperEventManager.callEvent(PaperEventManager.java:56) ~[luminol-1.21.1.jar:1.21.1-DEV-cc907e8]
        at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.callEvent(PaperPluginManagerImpl.java:131) ~[luminol-1.21.1.jar:1.21.1-DEV-cc907e8]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:628) ~[paper-mojangapi-1.21.1-R0.1-SNAPSHOT.jar:?]
        at net.minecraft.server.players.PlayerList.placeNewPlayer(PlayerList.java:440) ~[luminol-1.21.1.jar:1.21.1-DEV-cc907e8]
        at net.minecraft.server.network.ServerConfigurationPacketListenerImpl.lambda$handleConfigurationFinished$2(ServerConfigurationPacketListenerImpl.java:203) ~[luminol-1.21.1.jar:1.21.1-DEV-cc907e8]
        at io.papermc.paper.threadedregions.RegionizedTaskQueue$PrioritisedQueue$ChunkBasedPriorityTask.executeInternal(RegionizedTaskQueue.java:499) ~[luminol-1.21.1.jar:1.21.1-DEV-cc907e8]
        at io.papermc.paper.threadedregions.RegionizedTaskQueue$PrioritisedQueue.executeTask(RegionizedTaskQueue.java:429) ~[luminol-1.21.1.jar:1.21.1-DEV-cc907e8]
        at io.papermc.paper.threadedregions.RegionizedTaskQueue$RegionTaskQueueData.drainTasks(RegionizedTaskQueue.java:258) ~[luminol-1.21.1.jar:1.21.1-DEV-cc907e8]
        at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1696) ~[luminol-1.21.1.jar:1.21.1-DEV-cc907e8]
        at io.papermc.paper.threadedregions.TickRegions$ConcreteRegionTickHandle.tickRegion(TickRegions.java:407) ~[luminol-1.21.1.jar:1.21.1-DEV-cc907e8]
        at io.papermc.paper.threadedregions.TickRegionScheduler$RegionScheduleHandle.runTick(TickRegionScheduler.java:447) ~[luminol-1.21.1.jar:1.21.1-DEV-cc907e8]
        at ca.spottedleaf.concurrentutil.scheduler.SchedulerThreadPool$TickThreadRunner.run(SchedulerThreadPool.java:546) ~[luminol-1.21.1.jar:1.21.1-DEV-cc907e8]
        at java.base/java.lang.Thread.run(Thread.java:1583) ~[?:?]
```

**Comment by Benjiiben53** [comment]
Same problem with Paper 1.21.1 (Build #15) but the error is prompt every time we vanish or unvanished

**Comment by Syrent** [comment]
should be fixed in 1.5.0-SNAPSHOT-build.181
